deploy/kubernetes/elastic-agent-standalone-kubernetes.yml

```diff
@@ -87,9 +109,120 @@ data:
+        type: system/metrics
+        use_output: default
+        meta:
+          package:
+            name: system
+            version: 0.10.9
+        data_stream:
+          namespace: default
+        streams:
+          - data_stream:
+              dataset: system.core
+              type: metrics
+            metricsets:
+              - core
+            core.metrics:
+              - percentages
+          - data_stream:
+              dataset: system.cpu
+              type: metrics
+            period: 10s
+            cpu.metrics:
+              - percentages
+              - normalized_percentages
+            metricsets:
+              - cpu
+          - data_stream:
+              dataset: system.diskio
+              type: metrics
+            period: 10s
+            diskio.include_devices: null
+            metricsets:
+              - diskio
+          - data_stream:
+              dataset: system.filesystem
+              type: metrics
+            period: 1m
+            metricsets:
+              - filesystem
+            processors:
+              - drop_event.when.regexp:
+                  system.filesystem.mount_point: ^/(sys|cgroup|proc|dev|etc|host|lib|snap)($|/)
+          - data_stream:
+              dataset: system.fsstat
+              type: metrics
+            period: 1m
+            metricsets:
+              - fsstat
+            processors:
+              - drop_event.when.regexp:
+                  system.fsstat.mount_point: ^/(sys|cgroup|proc|dev|etc|host|lib|snap)($|/)
+          - data_stream:
+              dataset: system.load
+              type: metrics
+            period: 10s
+            metricsets:
+              - load
+          - data_stream:
+              dataset: system.memory
+              type: metrics
+            period: 10s
+            metricsets:
+              - memory
+          - data_stream:
+              dataset: system.network
+              type: metrics
+            period: 10s
+            network.interfaces: null
+            metricsets:
+              - network
+          - data_stream:
+              dataset: system.process
+              type: metrics
+            process.include_top_n.by_memory: 5
+            period: 10s
+            processes:
+              - .*
+            process.include_top_n.by_cpu: 5
+            process.cgroups.enabled: false
+            process.cmdline.cache.enabled: true
+            metricsets:
+              - process
+            process.include_cpu_ticks: false
+            system.hostfs: /hostfs
+          - data_stream:
+              dataset: system.process_summary
+              type: metrics
+            period: 10s
+            metricsets:
+              - process_summary
+            system.hostfs: /hostfs
+          - data_stream:
+              dataset: system.socket_summary
+              type: metrics
+            period: 10s
+            metricsets:
+              - socket_summary
+            system.hostfs: /hostfs
```

Comment:
This will be available starting 7.12, correct?

Reply:
👍🏼 That's correct!